client/gui/console.cpp

```diff
@@ -1,51 +1,42 @@
 #include "console.hpp"
 
-#include <string>
-#include <cstdio>
 #include <GLFW/glfw3.h>
+#include <cstdio>
+#include <string>
 
 namespace tec {
-	Console::Console() : buf{ new tec::RingBuffer< std::tuple< ImVec4, std::string >, 4096>() } {
+	Console::Console() : buf{ new tec::RingBuffer<std::tuple<ImVec4, std::string>, 4096>() } {
 		inputBuf[0] = '\0';
-		
+
 		// Default embed commands
-		this->AddConsoleCommand( "cmdlist",
-			"cmdlist : List all commands",
-		[this ] (const char*) {
+		this->AddConsoleCommand("cmdlist", "cmdlist : List all commands", [this](const char*) {
```

Comment:
these have a specific format

Reply:
Can you be a bit more specific? I can try to tweak clang format some, if I knew what you were referring to here for a "specific" format.

Reply:
I just think this is probably a case where an auto-formatting tool is going to struggle. The way these work at the moment, the command is on the first line, then the second line has the help, then there's a lambda after that. Can we force it to always preserve those commands?